main.go

```diff
@@ -73,9 +81,22 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	opts := zap.Options{
-		Development: true,
+
+	opts = zap.Options{
+		ZapOpts: []uberzap.Option{
+			uberzap.AddCaller(),
+			uberzap.AddStacktrace(uberzap.PanicLevel),
+		},
 	}
+
+	if developmentMode {
```

Comment:
Can we use `else` for the above declaration of `opts` here?

Reply:
I would prefer not to do that.

Currently the code reads like this 
```
set defaults
if dev_mode {
   override defaults
}
 ```
 
 Based on your suggestion, we can do either 
 ```
 if dev_mode {
    dev_mode settings
} else {
   default settings
}
```

OR 

 ```
 if !dev_mode {
    default settings
} else {
   dev_mode settings
}
```

In both the cases, the defaults get hidden inside the if/else block and I would prefer to have them outside. Let me know what you think..

Reply:
makes sense